manifest.json

```diff
@@ -4,7 +4,7 @@
 	"version_number": "#VERSION#",
 	"dependencies": [
 		"BepInEx-BepInExPack_ROUNDS-5.4.1100",
-		"willis81808-UnboundLib-2.1.3"
+		"willis81808-UnboundLib-2.9.0"
```

Comment:
Wrong version?

Reply:
Probably, but more-or-less irrelevant now since upcoming changes will require a currently unreleased version of Unbound